draft-ietf-tls-subcerts.md

```diff
@@ -275,8 +275,11 @@ mechanisms like proxy certificates {{?RFC3820}} for several reasons:
   which rely on a cryptographic binding between the entire certificate and the
   delegated credential, cannot.
 * Each delegated credential is bound to a specific signature algorithm for use
-  use in the TLS handshake ({{RFC8446}} section 4.2.3).  This prevents
+  in the TLS handshake ({{RFC8446}} section 4.2.3).  This prevents
   them from being used with other, perhaps unintended signature algorithms.
+  This prevents them from being used with other, perhaps unintended signature
+  algorithms. The signature algorithm bound to the delegated credential and
+  the one in the servers certificate can be different.
```

Comment:
This applies to more than just server certificates. Maybe say:
"The signature algorithm bound to the delegated credential can be chosen independently of the set of signature algorithms supported by the end-entity certificate."